express/blocks/columns/columns.css

```diff
@@ -120,4 +140,40 @@ main .columns .social.icon {
   main .columns.fullsize + h2 {
     margin-top: 0;
   }
+}
+
+@media (max-width: 899px) {
+  main .columns .hero-animation-overlay {
+    order: -1;
+  }
+
+  main .new-template-list-sixcols-container > div > .button-container {
```

Comment:
I'd use `.columns-fullsize-center-container` here to better keep blocks isolated from each other.